crates/wasi-provider/src/wasi_runtime.rs

```diff
@@ -156,6 +156,11 @@ impl WasiRuntime {
 
             for (key, value) in data.dirs.iter() {
                 let guest_dir = value.as_ref().unwrap_or(key);
+                info!(
```

Comment:
This should probably be `debug!` as it is something you probably don't care about unless debugging